lectures/supply_demand_multiple_goods.md

```diff
@@ -171,6 +191,57 @@ Verify that setting  $\mu=2$ in {eq}`eq:old3` also implies that formula
 
 ```
 
+```{code-cell} ipython3
```

Comment:
The class ``ExchangeEconomy``and its examples are shifted from lecture ``supply_demand_heterogeneity`` and simplified under the current setup.